views/system.ejs

```diff
@@ -73,28 +79,48 @@
+                                                                                                      href="https://github.com/NotExpectedYet/OctoFarm/releases"
+                                                                                                      target="_blank"><br><%= version %></a></p>
+                                                    <p class=""><i class="far fa-clock"></i> Timezone: <br><span class="text-light" id="timeZone" style="font-size:0.8rem;">
                                                          <%= systemInformation.sysUptime.timezoneName %> (<%= systemInformation.sysUptime.timezone %>)
-                                        </span>
+                                                    </span>
 
-                                            </p>
-                                            <p class=""><i class="fas fa-terminal"></i> Run Command: <br><span class="text-light" id="runCommand" style="font-size:0.8rem;">
+                                                    </p>
+                                                    <p class=""><i class="fas fa-terminal"></i> Run Command: <br><span class="text-light" id="runCommand" style="font-size:0.8rem;">
                                                                 <%= systemInformation.sysProcess.command %>
-                                        </span>
-
-                                            </p>
+                                                    </span>
 
+                                                    </p>
+                                                </div>
+                                                <div class="col-12 col-sm-12 col-md-6 col-lg-6 col-xl-6 text-center">
+                                                    <p><i class="fas fa-layer-group"></i> Service Information:
+                                                    <% if(serviceInformation.isDockerContainer !== 'undefined' && serviceInformation.isDockerContainer){ %>
+                                                        <p class="text-light"> <i class="fab fa-docker"></i></p>
+                                                    <% } %>
+                                                    <% if(serviceInformation.isNode !== 'undefined' && serviceInformation.isNode){ %>
+                                                        <p class="text-light">Powered by <i class="fab fa-node fa-2x"></i></p>
+                                                    <% } %>
+                                                    <% if(serviceInformation.update !== 'undefined' && serviceInformation.update.air_gapped !== 'undefined' && serviceInformation.update.air_gapped) { %>
+                                                        <p class="text-light"> <i class="fas fa-air-freshener"></i> Farm is air-gapped</p>
```

Comment:
Test for content? Can be mocked, needs some extra work in mocking the air-gapped boolean I guess

Reply:
Good point actually wanted to show connection available as an else as well.